src/v2/view-builder/views/email/ChallengeAuthenticatorDataEmailView.js

```diff
@@ -7,12 +7,20 @@ const BaseAuthenticatorEmailForm = BaseAuthenticatorEmailView.prototype.Body;
 const SubtitleView = View.extend({
   template: hbs`
     <div class="okta-form-subtitle" data-se="o-form-explain">
-      {{i18n
-        code="oie.email.verify.subtitle"
-        bundle="login"
-        arguments="email"
-        $1="<span class='strong no-translate'>$1</span>"
-      }}
+      {{#if email}}
+        {{i18n
+          code="oie.email.verify.subtitleWithEmailAddress"
+          bundle="login"
+          arguments="email"
+          $1="<span class='strong no-translate'>$1</span>"
+        }}
+      {{else}}
+        {{i18n
+          code="oie.email.verify.subtitleWithoutEmailAddress"
+          bundle="login"
+          $1="<span class='strong no-translate'>$1</span>"
```

Comment:
this one does not need `no-translate` class, since `your email` should be translated to all languages